src/github/client/github-installation-client.ts

```diff
@@ -198,10 +202,16 @@ export class GitHubInstallationClient {
 
 	public async getNumberOfReposForInstallation(): Promise<number> {
 		const response = await this.graphql<{ viewer: { repositories: { totalCount: number } } }>(ViewerRepositoryCountQuery);
-
 		return response?.data?.data?.viewer?.repositories?.totalCount;
 	}
 
+	public async getNumberOfReposForInstallationRest(): Promise<number> {
```

Comment:
Until we replace Probot, we can't make requests via graphql so, in the meantime, when testing out GHE server changes, we'll need to use the REST API.